lib/matplotlib/axes/_axes.py

```diff
@@ -1352,8 +1361,9 @@ def eventplot(self, positions, orientation='horizontal', lineoffsets=1,
         if len(linewidths) == 1:
             linewidths = np.tile(linewidths, len(positions))
         if len(colors) == 1:
-            colors = list(colors)
-            colors = colors * len(positions)
+            colors = list(colors) * len(positions)
+        if len(alpha) == 1:
+            alpha = list(alpha) * len(positions)
         if len(linestyles) == 1:
             linestyles = [linestyles] * len(positions)
```

Comment:
I'm also confused about the `[]` here; at this point, isn't `linestyles` a list? I guess it happens to work because the interior length-1 list gets looped by the `EventCollection`.

Reply:
I'm not clear on this as well, but decided that's not part of my business here. It could have something to do with linestyles supporting an on-off-sequence-tuple, which I think would be passed through to here, in which case you have to take care to repeat the tuple itself and not the seqence of elements in it.